users/permissions.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework import permissions
```

Comment:
You should do permissions adding in separate commit, splitting serializers in separate, and modifying/creating new views in separate.